### DISCUSSION TITLE
We are unable to complete your signup. Please read this guide for further details.

### DISCUSSION THREAD

**Discussion by sarowarhosen03** [vercel/community#5396] [Account Issue] [answered]
### Question

i couldnt login into vercel , at first im trying ot sing in with github it say my "your acccount has blocked" . then im try with my emial adrress it say "We are unable to complete your signup. Please read this guide for further details."
my email address is "tazuddin26@gmail.com"

**Comment by swarnava** [comment, marked as answer]
Hi,

You can email at registration@vercel.com.

**Reply by sarowarhosen03** [reply]
thanks
